components/authenticationHandler/loginRouteFactory.js

```diff
@@ -0,0 +1,38 @@
+const validateUser = require('./validateUser')
+
+module.exports = (options = {}) => {
+  const {
+    connection,
+    sessionCache,
+    log,
+  } = options
+
+  return async (request, response) => {
+    const {
+      username,
+      password,
+    } = request.body
+
+    if (await validateUser({
+      username,
+      password,
+      connection,
+    })) {
+      sessionCache.set(request.sessionID, true, (error, success) => {
+        if (!error && success) {
+          log.info(`Authenticated new session: ${request.sessionID}`)
+        }
+        else {
+          log.error(
+            `Error occured trying to cache session: ${request.sessionID}`
+          )
+        }
+      })
+      response.sendStatus('200')
+    }
+    else {
+      log.info('User could not be logged in.')
+      response.sendStatus('401')
```

Comment:
Maybe change the answer to 'confirmed' or 'denied' instead of an status code because 401 throws an error in the frontend and it can mislead to the assumption that the user is not authorized to perform the login which is wrong.